bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/digitalinvoice/view/DigitalInvoiceNavigationBarBottomAdapter.kt

```diff
@@ -0,0 +1,65 @@
+package net.gini.android.bank.sdk.capture.digitalinvoice.view
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.View.OnClickListener
+import android.view.ViewGroup
+import androidx.annotation.DrawableRes
+import net.gini.android.bank.sdk.databinding.GbsDigitalInvoiceNavigationBarBottomBinding
+import net.gini.android.capture.view.InjectedViewAdapter
+
+interface DigitalInvoiceNavigationBarBottomAdapter : InjectedViewAdapter {
+
+    fun setOnBackClickListener(listener: OnClickListener)
+
+    fun setOnHelpClickListener(listener: OnClickListener)
+
+    fun setOnPayClickListener(listener: OnClickListener)
+
+    fun setHelpButtonResource(@DrawableRes drawable: Int)
```

Comment:
[Request] We don't need this method. It's up to the implementation what drawable (if any) will be shown. In our default implementation we'll use our drawable, but clients could for example show a button with only text.